README.md

```diff
@@ -1,77 +1,108 @@
-# Organic Maps
 
-[![Google Play][google-play-badge]][google-play-url]
-[![App Store][apple-appstore-badge]][apple-appstore-url]
-[![Telegram Channel][telegram-channel-badge]][telegram-channel-url]
-[![Telegram Group][telegram-group-badge]][telegram-group-url]
-[![Build][build-badge]][build-url]
+# Organic Maps
 
 [Organic Maps](https://organicmaps.app) is an Android & iOS offline maps app
-for travellers, tourists, hikers and cyclists based on top of crowd-sourced
+for travelers, tourists, hikers and cyclists based on top of crowd-sourced
 OpenStreetMap data.
 
-![](docs/screenshots.jpg)
+<a href="https://testflight.apple.com/join/lrKCl08I">
+  <img src="docs/badges/apple-appstore.png" width="180">
+</a>
+<a href="https://play.google.com/store/apps/details?id=app.organicmaps">
+  <img src="docs/badges/google-play.png" width="180">
+</a>
+<a href="https://appgallery.huawei.com/#/app/C104325611">
+  <img src="docs/badges/huawei-appgallery.png" width="180">
+</a>
+<a href="https://f-droid.org/en/packages/app.organicmaps/">
+  <img src="docs/badges/fdroid.png" width="180">
+</a>
+
+## Screenshots
+
+<p float="left">
+  <img src="docs/screenshots/1.jpg" width="400" />
+  <img src="docs/screenshots/2.jpg" width="400" /> 
+  <img src="docs/screenshots/3.jpg" width="400" />
+  <img src="docs/screenshots/4.jpg" width="400" />
+</p>
 
 ## Features
 
-- Detailed Offline Maps for Travellers, Tourists, Hikers and Cyclists.
-- Contour Lines, Peaks, Passes, Sights, Toilets, ATMs, Parking.
-- Offline Car Navigation, Walking and Cycling Directions.
-- Open Source, Clean Code, No Spyware, No Ads, No Data Collection.
-- No Big Tech’s Prying Eyes on Your Private Trip Anymore.
+- Detailed offline maps with places missing on other maps
+- Cycling routes, hiking trails and walking paths
+- Contour lines, elevation profiles, peaks and slopes
+- Turn-by-turn walking, cycling and car navigation with voice guidance
+- Fast offline search through different categories e.g. restaurants, cafes, attractions, hotels, ATMs, etc. 
+- Dark mode to protect your eyes
+- Power saver mode to save the battery
+- Completely offline - no internet connection required
 
-## Installing
+## Privacy
 
-### Android
 
-- [Google Play][google-play-url]
-- [GitHub Releases APK][github-releases-url]
-- [Firebase App Distribution APK (Beta)][firebase-appdistribution-url]
+Organic Map are completely free from trackers and other bad stuff:
 
-### iOS
+- No ads
+- No tracking
+- No data collection
+- No phoning home
+- No annoying registration
+- No mandatory tutorials
+- No noisy email spam
+- No push notifications 
+- No crapware
+- <s>No pesticides</s> (just kidding)
```

Comment:
- ~~No pesticides~~ Pure and organic, made with love


- Respects your privacy
- Saves your battery
- No unexpected mobile data charges